filter_plugins/oo_filters.py

```diff
@@ -66,11 +66,25 @@ def oo_select_keys(data, keys):
 
   return retval
 
+def oo_max_nb_minions(groupname, groups, hostvars):
```

Comment:
We're really trying to avoid creating filters that are single purpose like this.

Instead of doing it this way, how about using the latest version of oo_collect and 'max' filter?

oo_collect returns a list of values based off the filters passed in, and 'max' gives the maximum value in the list.